backend/src/packages/projects/project.repository.ts

```diff
@@ -11,8 +11,16 @@ class ProjectRepository
     this.projectModel = projectModel;
   }
 
-  public async findAll(): Promise<ProjectEntity[]> {
-    const projects = await this.projectModel.query().execute();
+  public async findAll(id: number): Promise<ProjectEntity[]> {
```

Comment:
id of that?  findAll should return all
```suggestion
  public async findAll(): Promise<ProjectEntity[]> {
```

Reply:
for getting projects by userId I believe it should be `findByUserId` a separate method - so you don't need to change `findAll`